setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = qutip
+url = http://qutip.org
+description = QuTiP: The Quantum Toolbox in Python
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 2 - Pre-Alpha
```

Comment:
This is unrelated to this PR, but, are we still in  Pre-Alpha? That sounds strange given it's 4.5 etc. I suggest opening an issue to choose a dev status more adherent to reality.

Reply:
This only gets changed when we make a release - the "pre-alpha" status refers to the state of the master branch.